controllers/cloud.redhat.com/providers/kafka/appinterface.go

```diff
@@ -73,6 +73,8 @@ func validateKafkaTopic(ctx context.Context, cl client.Client, nn types.Namespac
 	t := strimzi.KafkaTopic{}
 	err := cl.Get(ctx, nn, &t)
 
+	// TODO: validate partitions and replicas?
```

Comment:
Validate in what way?

Reply:
If we think it's something worth checking, check that partitions/replicas on the KafkaTopic match what the user is requesting in their ClowdApp. It appears right now we only ensure that a topic with matching name exists.

Reply:
This is outside the scope for clowder - I think we assume that the thing in app-interface is right - I get that it means we will potentially "ignore" settings in the clowdapp - but I think this is OK for app-interface mode

Reply:
I'll drop the TODO